Makefile

```diff
@@ -1,8 +1,18 @@
 test: lint
 	@python3 -m unittest
 
-lint:
+lint: quotelint
 	@autopep8 -r --aggressive --diff --exit-code .
 	@flake8 --config=.flake8
 
+quotelint:
+	@x=$$(grep -rnH --include \*.py "\\\\[\"']");                  \
+	if [ "$$x" ]; then                                             \
+		echo "Please reconsider your choice of quoting for:";  \
+		echo "$$x";                                            \
+		exit 1;                                                \
+	fi >&2
+
```

Comment:
This seems to print:
```
Please reconsider your choice of quoting for:
test/test_model.py:57:        err_msg = "ERROR invalid value \"$2\" for option -r: relation not found"
test/test_model.py:95:        err_msg = "ERROR invalid value \"$2\" for option -r: relation not found"
test/test_model.py:665:        err_msg = "ERROR invalid value \"$2\" for option -r: relation not found"
test/test_framework.py:786:            self.assertEqual(str(e), 'attribute \'foo\' cannot be set to CustomObject: must be int/dict/list/etc')
Makefile:9: recipe for target 'quotelint' failed
```

I'm trying to figure out the wording, as the way you say 'please reconsider' sounds optional but exit 1 means its mandatory.
The highlighting of where the failing lines are is very nice.
I suppose this fails if you have a string with both ' and " in it, (I guess you could triple quote).

Reply:
I could change it to
```sh
echo "░█░█░█▀█░█░█░░░█▀█░█░█░█▄█░█▀█░▀█▀░█░█░░░░░░░█▀▀░▀█▀░█░█░░░█░█░█▀█░█░█░█▀▄░░░▄▀▄░█░█░█▀█░▀█▀░█▀▀░█▀▀░░░░\n░░█░░█░█░█░█░░░█░█░█░█░█░█░█▀▀░░█░░░█░░░░░░░░█▀▀░░█░░▄▀▄░░░░█░░█░█░█░█░█▀▄░░░█\\█░█░█░█░█░░█░░█▀▀░▀▀█░░▀░\n░░▀░░▀▀▀░▀▀▀░░░▀░▀░▀▀▀░▀░▀░▀░░░░▀░░░▀░░▄▀░░░░▀░░░▀▀▀░▀░▀░░░░▀░░▀▀▀░▀▀▀░▀░▀░░░░▀\\░▀▀▀░▀▀▀░░▀░░▀▀▀░▀▀▀░░▀░"
```

Reply:
>  I suppose this fails if you have a string with both ' and " in it, (I guess you could triple quote).

we already have this situation, and yes we triple quote---and that seems fine